crates/test-programs/wasi-http-tests/src/bin/outbound_request_response_build.rs

```diff
@@ -0,0 +1,40 @@
+use wasi_http_tests::bindings::wasi::{cli::stdout::get_stdout, http::types as http_types};
+
+fn print(msg: &[u8]) {
+    let _ = get_stdout().blocking_write_and_flush(&msg);
+}
```

Comment:
For the future this isn't necessary as `println!` should work in guests. If it doesn't that's a bug we should document and fix.

Reply:
If I use println, it gets translated to wasi preview1 but I wanted to exercise the preview2 functions

Reply:
Ah that's sort of right but also not quite accurate. It's correct that `println!` uses the Rust standard library which uses wasi preview1, but that's adapted to preview2 with the adapter, meaning that `println!` ends up calling the same preview2 function this one calls.

Reply:
But not in the core module, right?

Reply:
Sorry I'm not exactly sure what you mean, but when using `println!` the core module produced by rustc will refer to `wasi_snapshot_preview1::fd_write` if that's what you mean. That core import is then hooked up to the adapter, and the adatper imports preview2 instead.